build.gradle

```diff
@@ -3,9 +3,10 @@ buildscript{
     repositories {
         mavenCentral()
```

Comment:
You should be able to remove `mavenCentral()` because its contents are hosted in `jcenter()`

Reply:
Fixed in fe5b3d71a10967c157d09e9e39b63e6f876f2d29